home.html

```diff
@@ -0,0 +1,169 @@
+                </div>
+                <div class="main-program-card">
+                    <div class="left">
+                        <img src="./images/hotel reservation.png" alt="reservation-icon" />
+                        <p>Reservation</p>
+                    </div>
+                    <div class="right">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Doloribus corporis
+                    </div>
+                </div>
+                <div class="main-program-card">
+                    <div class="left">
+                        <img src="./images/travel insurance.png" alt="travel-insurance-icon" />
+                        <p>Travel Insurance</p>
+                    </div>
+                    <div class="right">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Doloribus corporis
+                    </div>
+                </div>
+                <div class="main-program-card">
+                    <div class="left">
+                        <img src="./images/airline ticket.png" alt="support-icone" />
+                        <p>Air-ticketing</p>
+                    </div>
+                    <div class="right">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    Doloribus corporis
+                    </div>
+                </div>
+                <div class="main-program-card">
+                    <div class="left">
+                        <img src="./images/currency.png" alt="workshop-icone" />
+                        <p>currency exchange</p>
+                    </div>
+                    <div class="right">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Doloribus corporis
+                    </div>
+                </div>
+            </div>
+            <div class="register" id="Join">
+                <a href="#">Register Today</a>
+            </div>
+        </section>
+        <section id="Team" class="Team"></section>
+        <section class="trustedby">
+            <h2 class="trustedby-title">Trusted By</h2>
+            <ul class="trustedby-ul">
+                <li><img src="images/rdb logo.jpg" alt="RDB-logo"></li>
+                <li><img src="https://cache.marriott.com/content/dam/marriott-digital/mc/global-property-shared/en_us/logo/assets/mc_logo_L.png" alt="marriot-logo"></li>
+                <li><img src="https://www.onomohotels.com/favicon.ico" alt="Onomo-logo"></li>
+                <li><img src="https://nezasafaris.com/wp-content/uploads/2021/05/logo.jpg" alt="neza-logo"></li>
+                <li><img src="images/RwandAir-Logo.wine.png" alt="rwandaair-logo"></li>
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <img src="images/logo.jpg" alt="logo" class="">
+      <h2 class="footer-p">&#169 2022 Created By Eligrand. All Rights Reserved</h2>
```

Comment:
- Please align the image to the left and the text to the right. It appears you centered them.